File: Frontend/src/app/models/brv-details.ts
import { JiraContentsExcel } from "./Jira/jira-contents-excel";

export class BrvDetails {
    idImpulse: string = '';
    creationDate: string = '';
    author: string = '';
    application: string = '';
    poApp: string = '';
    itApp: string = '';
    poDate: string = '';
    itDate: string = '';
    jiraContents: Array<JiraContentsExcel> = [];
}

File: Backend/Models/BrvDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Relmonitor.Models
{
    public class BrvDetails
    {
        public string IdImpulse { get; set; }
        public string CreationDate { get; set; }
        public string Author { get; set; }
        public string Application { get; set; }
        public string PoApp { get; set; }
        public string ItApp { get; set; }
        public string PoDate { get; set; }
        public string ItDate { get; set; }
        public List<JiraContentsExcel> JiraContents { get; set; }
    }
}
